codegen/testserver/generated_test.go

```diff
@@ -35,9 +35,60 @@ func TestForcedResolverFieldIsPointer(t *testing.T) {
 	require.Equal(t, "*testserver.Circle", field.Type.Out(0).String())
 }
 
+type testTracer struct {
+	id     int
+	append func(string)
+}
+
+func (tt *testTracer) StartOperationExecution(ctx context.Context) context.Context {
+	line := fmt.Sprintf("op:start:%d", tt.id)
+
+	tracerLogs, _ := ctx.Value("tracer").([]string)
+	ctx = context.WithValue(ctx, "tracer", append(tracerLogs, line))
```

Comment:
I think the tests are flaking out because appending to a slice isn't thread safe.

You now have multiple concurrent goroutines all working on the same underlying slice. You probably want something like:
`append(append([]string, tracerLogs...), line)`